src/couch/src/couch_server.erl

```diff
@@ -421,27 +425,37 @@ handle_call({open_result, T0, DbName, {ok, Db}}, {FromPid, _Tag}, Server) ->
                 true ->
                     Server#server.lru
             end,
-            {reply, ok, Server#server{lru = Lru}}
+            {reply, ok, Server#server{lru = Lru}};
+        [#entry{}] ->
+            % A mismatched opener pid means that this open_result message
+            % was in our mailbox but is now stale. Mostly ignore
+            % it except to ensure that the db pid is super dead.
+            exit(couch_db:get_pid(Db), kill),
```

Comment:
Will Db ever be undefined here, then maybe a catch around it?

Reply:
If it does then we have bigger problems as this is an {ok, Db} result we're handling specifically.

Reply:
Oh, derp, we are getting it from an opener, not grabbing from the ets table